.travis.yml

```diff
@@ -1,14 +1,16 @@
 language: go
 
 go:
-  - "1.12"
+  - "1.13"
+  - "1.14"
 
 go_import_path: github.com/kubeflow/tf-operator
 
 install:
   # get coveralls.io support
   - go get github.com/mattn/goveralls
   - curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $(go env GOPATH)/bin v1.18.0
+  - go mod vendor
```

Comment:
Any reason to vendor dependencies since this PR moves to go modules

Reply:
Just thought maybe easier to go build with vendor dir, if this is not needed, I'll delete it.

Reply:
I am fine to keep it now. Not a concern at this moment.